cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -146,11 +147,21 @@ func (c *Client) Sync() error {
 	}
 	defer cli.Close()
```

Comment:
The error return from the Close() calls is not logged.

Should the non-nil error be logged ?

Reply:
i don't think it will give us any value.